packages/types/src/insights.ts

```diff
@@ -97,7 +97,7 @@ export type TopBoard = {
     icon: string;
     title: string;
     activityCount: number;
-    activeUsers: string;
+    activeUsers: string[];
```

Comment:
absolutely nothing to change here, It is helpful to strict type this more by actually having this tied to the `UserProfile`'s id.

Reply:
Done.